qtfred/src/ui/widgets/sexp_tree.cpp

```diff
@@ -5177,6 +5182,43 @@ sexp_list_item* sexp_tree::get_listing_opf_asteroid_debris()
 	return head.next;
 }
 
+sexp_list_item* sexp_tree::get_listing_opf_lua_enum(int parent_node, int arg_index)
+{
+	int this_index = get_dynamic_parameter_index(tree_nodes[parent_node].text, arg_index);
+
+	// first child node
+	int child = tree_nodes[parent_node].child;
+	if (child < 0)
+		return nullptr;
+
+	if (this_index >= 0) {
+		for (int count = 0; count < this_index; count++) {
+			child = tree_nodes[child].next;
+		}
+	} else {
+		error_display(1,
+			"Expected to find an enum parent parameter for node %i in operator %s but found nothing!",
+			arg_index,
+			tree_nodes[parent_node].text);
```

Comment:
Just double checking that this doesn't need an early return?

Reply:
Honestly, it's been so long since I wrote this code that it's hard to remember for sure. But it does look like it. Added.